lang/de.xml

```diff
@@ -1,20 +1,265 @@
+				<load_complete>Fertig geladen.</load_complete>
+				<loading>Lade...</loading>
+				<load_saved_data>Diese Nutzerdaten laden?</load_saved_data>
+			</load>
+			<save name="Speichern">
+				<cancel_saving>Möchtest du das Speichern abbrechen?</cancel_saving>
+				<could_not_save>Konnte die Datei nicht speichern.
+Es fehlt freier Platz auf der Speicherkarte.</could_not_save>
+				<not_free_space>Es fehlt freier Platz auf der Speicherkarte.</not_free_space>
+				<new_saved_data>Neue Nutzerdaten</new_saved_data>
+				<saving_complete>Fertig gespeichert.</saving_complete>
+				<save_the_data>Diese Nutzerdaten speichern?</save_the_data>
+				<saving>Speichere...</saving>
+				<warning_saving>Speichere...
+Schalte die Konsole nicht aus und beende nicht die App.</warning_saving>
+				<overwrite_saved_data>Vorhandene Nutzerdaten überschreiben?</overwrite_saved_data>
+			</save>
+		</save_data>
+	</dialog>
+
 	<game_data>
-		<app_close>Folgende Anwendung wird geschlossen.</app_close>
+		<app_close>Die folgende App wird geschlossen.</app_close>
 	</game_data>
 
+	<indicator>
+		<app_added_home>Die App wurde dem Startbildschirm hinzugefügt.</app_added_home>
+		<delete_all>Lösche alle</delete_all>
+		<notif_deleted>Alle Benachrichtigungen werden gelöscht.</notif_deleted>
+		<install_failed>Installieren hat nicht geklappt.</install_failed>
+		<install_complete>Fertig installiert.</install_complete>
+		<installing>Installiere...</installing>
+		<no_notif>Es gibt keine Benachrichtigungen.</no_notif>
+		<trophy_earned>Du hast eine Trophäe freigeschaltet!</trophy_earned>
+	</indicator>
+
 	<initial_setup>
 		<back>Zurück</back>
-		<completed_setup>Die Erstinstallation ist jetzt abgeschlossen.
-Dein Vita3K-System ist bereit!</completed_setup>
-		<select_language>Wählen eine Sprache.</select_language>
+		<completed_setup>Du hast die Ersteinrichtung abgeschlossen.
+Dein Vita3K-System ist startklar!</completed_setup>
+		<select_language>Wähle eine Sprache.</select_language>
 		<next>Weiter</next>
 	</initial_setup>
 
 	<live_area>
 		<start>Starten</start>
-		<continue>Fortfahren</continue>
+		<continue>Weiter</continue>
 	</live_area>
+
+	<settings name="Einstellungen">
+		<theme_background name="Theme &amp; Hintergrund">
+			<default>Standard</default>
+			<theme name="Theme">
+				<information name="Informationen">
+					<name>Name</name>
+					<provider>Anbieter</provider>
+					<updated>Aktualisiert</updated>
+					<size>Größe</size>
+					<version>Version</version>
+				</information>
+				<delete>Dieses Theme wird gelöscht.</delete>
+			</theme>
+			<start_screen name="Startbildschirm">
+				<image>Bild</image>
+			</start_screen>
+			<home_screen_backgrounds>Hintergründe für Startbildschirm</home_screen_backgrounds>
+		</theme_background>
+		<date_time name="Datum &amp; Uhrzeit">
+			<date_format name ="Datums-Format">
+				<yyyy_mm_dd>JJJJ-MM-TT</yyyy_mm_dd>
+				<dd_mm_yyyy>TT.MM.JJJJ</dd_mm_yyyy>
+				<mm_dd_yyyy>MM/TT/JJJJ</mm_dd_yyyy>
```

Comment:
Need maintainer feedback about technical implications with these three vals.

Reply:
should based on psvita.

Reply:
Has been changed to the format on real Vita.